util/test/paratest.server

```diff
@@ -660,6 +660,19 @@ sub main {
 
     die "Error: CHPL_HOME must be set\n" unless defined $ENV{CHPL_HOME};
 
+    # Check for a compatible valgrind configuration
+    if ($valgrind > 0) {
+      $tasks = `../util/chplenv/chpl_tasks.py`; chomp $tasks;
```

Comment:
Think the path to util/chplenv should either be prefixed with `"$pwd"` or just use `$CHPL_HOME` like the call to printchplenv does